packages/btc/src/query/source.ts

```diff
@@ -63,6 +68,11 @@ export class DataSource {
     } as Utxo;
   }
 
+  async isTransactionConfirmed(hash: string): Promise<boolean> {
+    const tx = await this.service.getBtcTransaction(remove0x(hash));
```

Comment:
Sometimes a 404 error (ASSETS_API_RESOURCE_NOT_FOUND) can be thrown here without sufficient error context such as txid

![image](https://github.com/ckb-cell/rgbpp-sdk/assets/1297478/1bf3451d-432d-4682-b006-489a7cecd1ee)

Reply:
Any suggestions on how to provide clearer context? A simple solution is that we can inject the request context into the error, which is shown in the following code. However, normally this method doesn't provide great TypeScript support.
```ts
try { 
  ... 
} catch (e: any) {
  if (e instanceof BtcAssetsApiError) {
    e.context = { txid };
  }
} 
```

Reply:
LGTM.
With a txid, the develop using SDK may be clearer to track the error.

Feel free to implement in another PR, maybe other errors need this kind of context too.